Sources/L10nModels/LocalizedStringFile.swift

```diff
@@ -53,40 +65,43 @@ struct LocalizedStringFile {
 }
 
 extension LocalizedStringFile {
-    func parser() -> Parser<[LocalizedStringEntry]> {
-        let comment = zip(
-            literal("/* "),
-            string(until: literal(" */")),
-            literal(" */"),
-            zeroOrMoreSpacesOrLines
+    public static func parser() -> Parser<[LocalizedStringEntry]> {
+        let comment = Parser.zip(
+            .literal("/* "),
+            .string(until: .literal(" */")),
+            .literal(" */"),
+            .zeroOrMoreSpacesOrLines
         ).map { _, comments, _, _ in comments }
 
-        let localizedStringEntry: Parser<LocalizedStringEntry> = zip(
+        let localizedStringEntry: Parser<LocalizedStringEntry> = Parser.zip(
             comment,
-            literal("\""),
-            string(until: literal("\" = \"")),
-            literal("\" = \""),
-            string(until: literal("\";")),
-            literal("\";"),
-            zeroOrMoreSpacesOrLines
+            .literal("\""),
+            .string(until: .literal("\" = \"")),
+            .literal("\" = \""),
+            Parser.string(until: .literal("\";")).replaceMismatch(with: ""),
```

Comment:
This is the fix for the empty string. When parser .string(until:) returns empty, it "fails". We recover to empty string.
The next literal "; still needs to match anyway, so it's ok to recover from this one, as long as eventually the string gets closed.